environments/vm/group_vars/vm.yml

```diff
@@ -14,28 +14,28 @@ error_mail_to: devnull@example.org
 
 php_display_errors: 1
 
-attribute_aggregation_gui_version: "2.1.1"
-attribute_aggregation_server_version: "2.1.1"
-oidc_playground_client_version: "2.4.3"
-oidc_playground_server_version: "2.4.3"
-engine_version: "6.6.5"
-manage_gui_version: "6.0.10"
-manage_server_version: "6.0.10"
-lifecycle_version: "0.0.5"
+attribute_aggregation_gui_version: "2.1.2"
+attribute_aggregation_server_version: "2.1.2"
+oidc_playground_client_version: "3.0.0"
+oidc_playground_server_version: "3.0.0"
+engine_version: "6.7.0"
+manage_gui_version: "7.0.0"
+manage_server_version: "7.0.0"
+lifecycle_version: "0.1.1"
 monitoring_tests_version: "6.1.1"
 mujina_version: "7.4.4"
-oidcng_version: "5.0.5"
-pdp_server_version: "3.0.1"
-pdp_gui_version: "3.0.1"
-profile_version: "3.0.3"
-teams_gui_version: "8.4.3"
-teams_server_version: "8.4.3"
-voot_version: "4.0.2"
-myconext_server_version: "5.1.7"
-myconext_gui_version: "5.1.7"
-account_gui_version: "5.1.7"
-dashboard_server_version: "10.1.5"
-dashboard_gui_version: "10.1.5"
+oidcng_version: "6.0.1"
+pdp_server_version: "4.0.0"
+pdp_gui_version: "4.0.0"
+profile_version: "3.0.4"
+teams_gui_version: "8.5.3"
+teams_server_version: "8.5.3"
```

Comment:
This is still the java 8 version. The fix that is in 8.5.3 is still to be ported to the 9.x branch (which is the Java11 version).

I'll update these versions when that one is ready